sdk/identity/azure-identity-broker/src/main/java/com/azure/identity/broker/package-info.java

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+/**
+ * <p>The Azure Identity Brokered Authentication library offers support for using various support brokers from the
+ * MSAL4j library.</p>
```

Comment:
Docs: This needs to be updated to have detailed docs. azure-identity can be used as reference.